index.html

```diff
@@ -2092,6 +2092,13 @@ <h3>Data Schemas</h3>
           </dd>
         </dl>
 
+        <p class="note">
+The <code>credentialSchema</code> provides an opportunity to annotate type
+definitions or lock them to specific versions of the vocabulary. Authors of VCs
+can include a static version of their vocabulary via credentialSchema that is
+locked to some content integrity protection mechanism.
```

Comment:
You may also want to make a point about doing syntactic checking on the credential as well using stuff like JSON Schema. Just reminding folks that this property has at least two uses.

Reply:
In any case, approve of change whether or not you take the above into account.

Reply:
Made some changes  in [cd62b44](https://github.com/w3c/vc-data-model/pull/533/commits/cd62b44daf6b6767d8eee86280c948096668de61)